www/js/app.js

```diff
@@ -1143,6 +1166,11 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 if (docBody) {
                     docBody.addEventListener('dragover', handleIframeDragover);
                     docBody.addEventListener('drop', handleIframeDrop);
+                    docBody.addEventListener('keydown',function(e) {
+                        // if cursor is not inside a input element only then focus #prefix
+                        if (e.target.nodeName !== 'INPUT')
```

Comment:
If we put an option for this feature, I suggest to not try to cover its side effects : it's probably impossible to achieve

Reply:
New commit follows this comment.